test/python/circuit/library/test_xx_minus_yy.py

```diff
@@ -0,0 +1,108 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test XXMinusYYGate."""
+
+import unittest
+import numpy as np
+from scipy.linalg import expm
+from ddt import data, ddt, unpack
+
+from qiskit import QuantumCircuit
+from qiskit.circuit.library import RZGate, XGate, YGate
+from qiskit.quantum_info import Operator
+from qiskit.circuit.library import XXMinusYYGate
+
+
+@ddt
```

Comment:
These kinds of tests are common for the standard gates, we probably don't need this file and can move the tests into the already existing test files:
* the inverse test should already be covered by https://github.com/Qiskit/qiskit-terra/blob/e84462db06aae8e7a066f5cc5281455014259217/test/python/circuit/test_gate_definitions.py#L219, so I think that can be removed
* the decomposition tests are here: https://github.com/Qiskit/qiskit-terra/blob/e84462db06aae8e7a066f5cc5281455014259217/test/python/circuit/test_gate_definitions.py#L74
* the `to_matrix` test is sort of covered here: https://github.com/Qiskit/qiskit-terra/blob/e84462db06aae8e7a066f5cc5281455014259217/test/python/circuit/test_extensions_standard.py#L1365 for one set of parameters. If you want you could add more cases for this gate there.

This being said, these tests could use a cleanup or rename (especially the `test_extensions_standard` file since all the gates don't live in `qiskit.extensions.standard` anymore!), but let's keep that for a separate PR.

Reply:
The name of the test described the module where the standard gates used to live: `qiskit.extensions.standard` --> `test_extensions_standard`. We moved them to `qiskit.circuit.library` in #4035 didn't yet change the name of the test file.